Enable mongodb for labs service

DvrkRain opened the pull request:
- [x\]  Established connection to MongoDB
- [x\] Completed `Dockerfile` and `docker-compose.yml` for local testing
- [x\] Moved `requirements.txt` file into app folder
- [x\] Removed "imports fix" in `Dockerfile.labs-service` due to its useless

DvrkRain committed:
feat(labs): added tester for `CreateLab` client function

DvrkRain committed:
feat(labs): fixed `models` import typo; Added modules to requirements

DvrkRain committed:
feat(labs): completed tester

DvrkRain committed:
feat(articles): removed unused and irrelevant imports; fixed `.proto` files import; configured logging; created methods descriptions; fixed bug that `DownloadAsserResponse` doesn't receive metadata first; removed irrelevant `encode-decode` actions;

DvrkRain committed:
feat(articles): changed .env and its example; established config logging; completed working dockerfile and docker-compose files; fixed download asset logging bug

DvrkRain committed:
feat(articles): removed non-important module

DvrkRain committed:
feat(articles): removed irrelevant statement in dockerfile (imports fixed); added logging to see if some method was called

DvrkRain committed:
feat(labs): established connection to MongoDB; configured logging to `config.py`; completed `Dockerfile` and `docker-compose.yml` for local testing

DvrkRain committed:
feat(labs): established connection to MongoDB; configured logging to `config.py`; completed `Dockerfile` and `docker-compose.yml` for local testing

DvrkRain committed:
feat(labs): moved `requirements.txt` file into app folder; removed "imports fix" in `Dockerfile` due to useless;

DvrkRain committed:
feat(labs): moved `requirements.txt` file into app folder; removed "imports fix" in `Dockerfile` due to useless;

lanebo1 reviewed (approved):
Sounds like OK

DvrkRain committed:
feat(articles): hotfix on article service environment variables

Nexonm reviewed (approved):
Sounds like ok. Hate main.py file for it's size.

DvrkRain committed:
Merge branch 'dev' into 174-create-tests-for-labs-service

DvrkRain committed:
Create tests for labs service

Merge pull request #185 from IU-Capstone-Project-2025/174-create-tests-for-labs-service

DvrkRain committed:
Merge branch '197-enable-mongodb-for-labs-service' of https://github.com/IU-Capstone-Project-2025/open-labs-share into 197-enable-mongodb-for-labs-service
